createInvoice/src/main/resources/templates/bulk/displayBulk.html

```diff
@@ -0,0 +1,82 @@
+<div id="page" layout:fragment="content" layout:decorator="layouts/default">
+	<div class="container" style="padding:0;margin:0px; margin-bottom:10px;">
+		
+		
+		<div class="row" style="padding-top:50px; ">
+			<div class="col-sm-4">
+				<h3>Bulk call initiated with ID <span th:text="${bulkStatus.id}" /> </h3>
+				<input type="hidden" th:value = "${ bulkStatus.id }" id="bulkId"/> 
+				<input type="hidden" th:value = "${ selectedObject }" id="selectedObject"/> 
+				
+				This bulk is currently in status <b th:text="${bulkStatus.status}" />. 
+				
+				This page is launching status verification calls every minute and will download the results once the bulk file generation has been completed.		
+			</div>
+			
+			<div id="statusReport" style="display:none;">
+				Fetching bulk status from Cloud Elements
+			</div>
+			
+		</div>
+		
+		
+		<div class="row" style="padding-top:25px">
+			<div class="col">
+				<button type="submit" class="btn btn-primary" style="width:100%"
+				onclick="">List meta data <span class="fa fa-next" /></button>
+			</div>
+		</div>
+	</div>
+	
+	
+	<script language="javascript">
```

Comment:
It would be nice to break out sizable javascript chunks into their own files

Reply:
Keeping this in for now, Bulk is the next project I am working on. Not fully done yet but did not want to split off yet from current codebase